src/partials/settings_sections.html

```diff
@@ -45,6 +45,13 @@ <h2 class="text-gray-900 font-light text-center text-xl md:text-3xl">
             </span>
           </span>
         </div>
+        <label class="text-sm text-gray-700 font-medium mt-1">
+          <input
+          data-el="livebook-desktop-toggle"
+          class="mr-1"
```

Comment:
To make it consistent across browsers, let's add some universal styles as we do in Livebook [here](https://github.com/livebook-dev/livebook/blob/78b9a11d8c3dac78c648cc7903343ea09f45efd9/assets/css/components.css#L166-L177). You can copy that to `components.css` :)

Reply:
done 👍